pkg/kubelet/kuberuntime/fake_kuberuntime.go

```diff
@@ -0,0 +1,482 @@
+/*
+Copyright 2016 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package kuberuntime
```

Comment:
Can we put the `fake_` files here in their own package that's only referenced from tests (e.g. `kuberuntime/testing`) so that they don't get compiled into the final kubelet binary?